opendatahub-io/kubeflow
# Add Support for Cluster Proxy

## Description


## How Has This Been Tested?
- OCP cluster 4.11
- Update `cluster` Proxy resource to resemble -
```
apiVersion: config.openshift.io/v1
kind: Proxy
metadata:
  name: cluster
spec:
  httpProxy: http://<username>:<pswd>@<ip>:<port> 
  httpsProxy: http://<username>:<pswd>@<ip>:<port> 
  noProxy: example.com 
  readinessEndpoints:
  - http://www.google.com 
  - https://www.google.com
  trustedCA:
    name: user-ca-bundle 
```
- Deploy live-build : quay.io/modh/rhods-operator-live-catalog:1.20.0-rhods-5811
- Start Jupyter Server
- Verify:
      -  if `trusted-ca` configmap is created in user namespace
      - configmap is mounted in Notebook resource
      - proxy Env variables are added to Notebook pod
## Merge criteria:
